solc/CommandLineInterface.h

```diff
@@ -101,7 +101,9 @@ class CommandLineInterface
 	void handleBinary(std::string const& _contract);
 	void handleOpcode(std::string const& _contract);
 	void handleIR(std::string const& _contract);
+	void handleIRAst(std::string const& _contractName);
```

Comment:
```suggestion
	void handleIRAst(std::string const& _contract);
```

Just to be consistent with the other methods, but maybe it would be better to change all others to `_contractName` as well. Not sure if we should do it in this PR though.